bundle/Resources/translations/messages.en.yml

```diff
@@ -1 +1,2 @@
-field_definition.enhancedbinaryfile.allowedTypes: Allowed file types (comma separated)
+field_definition.enhancedbinaryfile.allowedTypes: Allowed file types (pipe-separated)
+field_definition.enhancedbinaryfile.mimeTypesMessage: Wrong mimetype error message (https://symfony.com/doc/current/reference/constraints/File.html#mimetypesmessage)
```

Comment:
This looks like a placeholder?

Reply:
Hey @emodric ,
actually  it's not a placeholder. I thought it would be a good idea to include the link to symfony's documentation in the help text for the user to understand that placeholders can be used in the specified error message. Do you advice to not do it tha way?

Reply:
I'd put a default Symfony message that already has some placeholders, and put a comment in the YML file explainging that you can use placeholders and linking to the page above

Reply:
I think there's a missunderstanding here. We're talking about the *label* for the input field on content type level, here. That label will read `Wrong mimetype error message (https://symfony.com/doc/current/reference/constraints/File.html#mimetypesmessage)`. 

The default error message is unchanged, it will still be symfony's default error message which can be found under said link and it will also contain placeholders.

Reply:
Ah, okay then :)